crates/host/src/wasmbus/mod.rs

```diff
@@ -2918,7 +2920,14 @@ impl Host {
                         .exporter_otlp_endpoint
                         .clone()
                         .unwrap_or_default(),
-                )
+                );
+
+            if cfg!(windows) {
+                // Proxy SYSTEMROOT to providers. Without this, providers on Windows won't be able to start
+                child_cmd.env("SYSTEMROOT", env::var("SYSTEMROOT").unwrap_or_default());
```

Comment:
Thoughts on unwrapping to the default of `C:\\Windows` just in case it's not found for whatever reason to _hopefully_ catch that case for the majority of users?

Reply:
I think bailing works just fine and we can encourage users to set `SYSTEMROOT` if they for some reason do not have it set.

Reply:
Updated!